src/main/content/img/guide_lightningbolt.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
At some point the images from the design team will be delivered optimized. In the meantime it's best to optimize them before being added to the repository. That way the repo size is not negatively and unnecessarily impacted and we don't have to add extra complexity to the build process, making it take longer than it should.